### DISCUSSION TITLE
Gulp Sass Compile Error

### DISCUSSION THREAD

**Discussion by Axos11** [bootscore/bootscore#392] [General]
I know, that most of the people use the integrated scss compiler but as it is the only error or warning I get while compiling we could maybe address it anyways ;)
 
Message: ..\www\wp-content\themes\bootscore-main\scss\bootscore_woocommerce\_wc_qty_btn.scss
Error: compound selectors may no longer be extended.
Consider `@extend .form-control, :disabled` instead.

See https://sass-lang.com/d/extend-compound for details.

**Comment by crftwrk** [comment]
Moved to issues https://github.com/bootscore/bootscore/issues/393
